lib/timber/integrations/phoenix_instrumenter.ex

```diff
@@ -72,9 +72,98 @@ defmodule Timber.Integrations.PhoenixInstrumenter do
   require Logger
 
   alias Timber.Event
+  alias Timber.Events.ChannelJoinEvent
+  alias Timber.Events.ChannelReceiveEvent
   alias Timber.Events.ControllerCallEvent
   alias Timber.Events.TemplateRenderEvent
 
+  #
+  # Channels
+  #
+
+  @doc false
+  @spec phoenix_channel_join(:start | :stop, map | non_neg_integer, map | :ok) :: :ok
+  def phoenix_channel_join(:start, _compile, %{socket: socket, params: params}) do
+    log_level = get_log_level(:info)
+    channel = try_atom_inspect(socket.channel)
+    topic = socket.topic
+    transport = try_atom_inspect(socket.transport)
+    serializer = try_atom_inspect(socket.serializer) # atom (module name) | nil
```

Comment:
I think that the comment here is meant to suggest that `socket.serializer` will return either the module name as an atom or the value `nil`. It may be more appropriate to hoist this comment to above `channel = …` as clarification for all the assignments.